doc/2/api/controllers/security/check-rights/index.md

```diff
@@ -0,0 +1,93 @@
+---
+code: true
+type: page
+title: checkRights
+---
+
+# checkRights
+
+<SinceBadge version="change-me"/>
```

Comment:
Is this normal?

Reply:
Yes, the release tool will replace this by the correct version number when creating the release